File: URI/notas_e_moedas.js

function preenche(){
	var cem = 0
	var cinquenta= 0
	var vinte= 0
	var dez= 0
	var cinco= 0
	var dois= 0
	var umR= 0
	var cinquentaC= 0
	var vinteCincoC= 0
	var dezC= 0
	var cincoC= 0
	var umC= 0
	var D = 576.76
	D = parseFloat(toFixed(D,2))
	if(D >= 0 & D <= 1000000.00){
		while(D != 0 ){
			while (D > 100.00) {
				cem ++
				D-=100.00
			}
			while (D > 50.00) {
				cinquenta ++
				D-=50.00
			}
			while (D > 20.00) {
				vinte ++
				D-=20.00
			}
			while (D > 10.00) {
				dez ++
				D-=10.00
			}
			while (D > 5.00) {
				cinco ++
				D-=5.00
			}
			while (D > 2.00) {
				dois ++
				D-=2.00
			}
			while (D > 1.00) {
				umR ++
				D-=1.00
			}
			while (D > 0.50) {
				cinquentaC ++
				D-=0.50
			}
			while (D > 0.25) {
				vinteCincoC ++
				D-=0.25
			}
			while (D > 0.10) {
				dezC ++
				D-=0.10
			}
			while (D > 0.05) {
				cincoC ++
				D-=0.05
			}
			while (D > 0.01) {
				umC ++
				D-=0.01
			}
			break;
		}
	}
console.log(`NOTAS:
${cem} nota(s) de R$ 100.00
${cinquenta} nota(s) de R$ 50.00
${vinte} nota(s) de R$ 20.00
${dez} nota(s) de R$ 10.00
${cinco} nota(s) de R$ 5.00
${dois} nota(s) de R$ 2.00
MOEDAS:
${umR} moeda(s) de R$ 1.00
${cinquentaC} moeda(s) de R$ 0.50
${vinteCincoC} moeda(s) de R$ 0.25
${dezC} moeda(s) de R$ 0.10
${cincoC} moeda(s) de R$ 0.05
${umC} moeda(s) de R$ 0.01`)
}

File: URI/notas_e_moedas.py
g = float(input())
D = float("{0:.2f}".format(g))
cem = 0
cinquenta= 0
vinte= 0
dez= 0
cinco= 0
dois= 0
umR= 0
cinquentaC= 0
vinteCincoC= 0
dezC= 0
cincoC= 0
umC= 0
if(D >= 0) and (D <= 1000000.00):
    while(D > 0.00 ):
    	while (D >= 100.00):
    		cem +=1
    		D-=100.00;
    	while (D >= 50.00):
    		cinquenta +=1
    		D-=50.00;
    	while (D >= 20.00):
    		vinte +=1
    		D-=20.00;
    	while (D >= 10.00):
    		dez +=1
    		D-=10.00;
    	while (D >= 5.00):
    		cinco+=1
    		D-=5.00;
    	while (D >= 2.00):
    		dois +=1
    		D-=2.00;
    	while (D >= 1.00):
    		D-=1.00;
    		umR +=1
    	while (D >= 0.50):
    		D-=0.50;
    		cinquentaC +=1
    	while (D >= 0.25):
    		D-=0.25;
    		vinteCincoC +=1
    	while (D >= 0.10):
    		D-=0.10;
    		dezC +=1
    	while (D >= 0.05):
    		D-=0.05
    		cincoC +=1
    	umC=D*100;
    	break
print(D)
print("NOTAS:")
print("%i nota(s) de R$ 100.00" %(cem))
print("%i nota(s) de R$ 50.00" %(cinquenta))
print("%i nota(s) de R$ 20.00" %(vinte))
print("%i nota(s) de R$ 10.00" %(dez))
print("%i nota(s) de R$ 5.00" %(cinco))
print("%i nota(s) de R$ 2.00" %(dois))
print("MOEDAS:")
print("%i moeda(s) de R$ 1.00" %(umR))
print("%i moeda(s) de R$ 0.50" %(cinquentaC))
print("%i moeda(s) de R$ 0.25" %(vinteCincoC))
print("%i moeda(s) de R$ 0.10" %(dezC))
print("%i moeda(s) de R$ 0.05" %(cincoC))
print("%i moeda(s) de R$ 0.01" %(umC))